spec/services/resource_recommendations_spec.rb

```diff
@@ -1,21 +1,19 @@
 describe ResourceRecommendations do
-    let(:moment) {FactoryBot.build(:moment,
-                                       name: 'SELF-INJURY@',
-                                       why: 'text',
-                                       fix: 'text' )}
+  let(:moment) { FactoryBot.build(:moment, name: 'SELF-INJURY@', why: 'text', fix: 'text' ) }
+  let(:current_user) { create(:user2) }
+  subject(:resources) { ResourceRecommendations.new(moment: moment, current_user: current_user).call}
 
-    subject(:resources) { ResourceRecommendations.new(moment).call}
+  describe 'A test for resources method from ResourceRecommendations class' do
```

Comment:
Could we add a spec for the crisis prevention modal, I believe this feature will grow.

Reply:
Yes that is a good idea! I'll do that!